packages/objectloader/src/index.js

```diff
@@ -576,15 +576,18 @@ export default class ObjectLoader {
       return {}
     }
 
-    const store = this.cacheDB
-      .transaction('objects', 'readwrite')
-      .objectStore('objects')
-    for (const obj of objects) {
-      const idAndData = obj.split('\t')
-      store.put(idAndData[1], idAndData[0])
+    try {
+      const store = this.cacheDB
+        .transaction('objects', 'readwrite')
+        .objectStore('objects')
+      for (const obj of objects) {
+        const idAndData = obj.split('\t')
+        store.put(idAndData[1], idAndData[0])
+      }
+      return this.promisifyIdbRequest(store.transaction)
+    } catch (e) {
+      this.logger.error(e)
```

Comment:
There's no longer a return in the function in the error scenario. Even if `undefined`/`void` is fine, we should be explicit about it.